<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminstrator</class>
 <widget class="QWidget" name="Adminstrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>848</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="accountView" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Search account</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchAccountBox"/>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>* Please right click at row to change the account informations</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="usersTableView">
         <property name="styleSheet">
          <string notr="true">QTableView {
border: 1px solid rgba(118, 122, 142, 153);
background: rgba(224, 222, 225, 51);
selection-background-color: rgba(136, 132, 159, 204);
selection-color: rgb(248, 246, 250);
}

QHeaderView::section {
border-radius: 30px;
height: 30px;
color: rgb(239, 238, 241);
background: rgb(108, 104, 124);
}
</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::CurrentChanged</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bookView" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item alignment="Qt::AlignLeft">
              <widget class="QFrame" name="frame_5">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Search book</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchBookBox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Library management</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="bookTableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="buttonWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="addBookBtn">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Add Book</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="deleteBookBtn">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Delete Book</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="accountManagementBtn">
        <property name="text">
         <string>Account Management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bookManagementBtn">
        <property name="text">
         <string>Book Management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backToLoginBtn">
        <property name="text">
         <string>Back to Log In</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
